sdk_lib/make_chroot.sh

```diff
@@ -154,6 +159,13 @@ init_setup () {
    ln -sf "${CHROOT_TRUNK_DIR}/src/third_party/portage-stable" \
      "${FLAGS_chroot}"/"${PORTAGE_STABLE_OVERLAY}"
 
+   if [[ -n "${FLAGS_extra_portdir_overlay_path}" ]]; then
+     if [[ -n "${FLAGS_extra_portdir_overlay_name}" ]]; then
+       ln -sf "${CHROOT_TRUNK_DIR}"/"${FLAGS_extra_portdir_overlay_path}" \
```

Comment:
Actually, or just not bother with a symlink at all. Just make sure the overlay path is somewhere in the source tree (and thus accessible in the chroot) and translate the path to the right location under /mnt/host before passing it to update_chroot.